### ISSUE TITLE
Studio Library not capturing and displaying thumbnails

### ISSUE THREAD

**Issue by danaoconnor** [krathjen/studiolibrary#367]
Studio Library suddenly stopped capturing thumbnails and displaying them. It goes through the process and I can see the thumbnails being captured but when it is finished there are no images. No sequence folder is created.

Anyone else having this problem?

**Comment by matrees00** [comment]
Yes, I started having it recently.  The animation is definitely saved because its in the folder, but it's not showing up in studio library.
Any fix suggestions welcome!

**Comment by danaoconnor** [comment]
We've found no solution yet. This has happened before but the problem went away as mysteriously as it came. Those times were only for a couple days, this time it's been a couple of months. And, it's just me. For everyone else it is working as expected.

We are going to try  copying files to a temp directory and then uninstalling/reinstalling Studio Library, and move files back to original folders. Been too busy to try but that is our next step.

**Comment by mrpoddar** [comment]
I have faced this issue many times. Debug mode shows Access Denied to the Windows Temp folder. Reinstalling StudioLib fixed this for me the last time it happened.

**Comment by matrees00** [comment]
Hi, we managed to fix this yesterday.  In my case it was to do with the number of sub-folders we had in Studio Library.  it was set to 3 so anything more than that and the animation thumbnail would not display.  You can change the folder depth setting in the config file which should be saved wherever your studio library folder is.  Increasing the number fixes it but potentially slows the tool down.

Hope that helps.

**Comment by danaoconnor** [comment]
I've looked everywhere I can think of and I can't find a config file of any kind. Where did you find yours and what was it called?

**Comment by matrees00** [comment]
For me there's a config folder in the studio library folder which contans a config.json file.  I have no idea where else it may be if not there sorry.

**Comment by LushMush** [comment]
Hey so I ran into this today and spent a few hours trying to fix it. 
-I temporarily renamed my preferences folder to force maya into creating a new preferences folder, this did not help. 
-I reinstalled studio library entirely, it did not help.
-I made sure my temp folder has all permissions allowed, still no luck. 

On a whim, i tried clearing out my whole temp folder. This fixed it! I'm thinking there may have been some file in the temp folder that was having issues being overwritten. To do this on windows, navigate to your temp folder (type %tmp% into your windows directory and press enter), select everything in there (or just the broken file/folder in question if you'd prefer), and send it to the trash bin. Clear your trash bin just to be safe.
